packaging/protobuf.spec

```diff
@@ -5,7 +5,9 @@ Release:        1%{?dist}
 License:        BSD
 Group:          Development/Libraries
 Source:         https://github.com/google/protobuf/releases/download/v%{version}/protobuf-all-%{version}.tar.gz
-Source1001:     protobuf.manifest
+#Source1:        proto.vim
+#Patch3:         0001-Add-generic-GCC-support-for-atomic-operations.patch
+#Patch4:         protobuf-2.5.0-makefile.patch
```

Comment:
Is it safe to remove manifest for Tizen systems? It usually makes troubles in run time.